appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -263,6 +263,15 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
         }
     }
 
+    /**
+     *  Provide the APNs device token to Appcues.
+     *
+     *  @param token A globally unique token that identifies this device to APNs.
```

Comment:
should update this comment to reflect Firebase Cloud Messaging (FCM) rather than APNs